### PR TITLE
Renames brown gas and stimulum

### PR DESCRIPTION
**Pull request by nicbn**
[Changelogs]:
:cl: nicbn
tweak: Renamed brown gas to nitro dioxide.
tweak: Renamed stimulum to nikolayev gas.
/:cl:

The main reason is that brown's gas is the name of a gas that is not nitrogen dioxide.

### CODE REVIEW

**Comment by jughu** [comment]
WHY DOES EVERYTHING NEED TO SOUND COMPLICATED
WHATS THE NEXT GAS
WUBBAWUBBADADUBDUB?

**Comment by nicbn** [comment]
no rick and morty allowed in my pr

**Comment by praisenarsie** [comment]
@as334 

also the n2o rename is fake and gay.

**Comment by duncathan** [comment]
this is terrible LMAO

**Comment by duncathan** [comment]
please do the canister icons in a separate PR because those are good and will likely be merged

**Comment by nicbn** [comment]
The main reason is that brown's gas is a name to ANOTHER GAS that is not nitrogen dioxide.
N2O rename is to avoid confusion, and is just the removal of the "oxide" word, so stop.
Nikolayev one is one reference to the russian experiment myth.

**Comment by nicbn** [comment]
If the problem is N2O I can keep it as is and rename the other ones only.

File: code/modules/atmospherics/gasmixtures/gas_types.dm [hunk 1]
```diff
@@ -79,16 +79,16 @@ GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /
 /datum/gas/nitrous_oxide
 	id = "n2o"
 	specific_heat = 40
-	name = "Nitrous Oxide"
+	name = "Nitrous"
```
Comment (Cyberboss) [inline code/modules/atmospherics/gasmixtures/gas_types.dm]: Nitrous Oxide is ss13 history and I'm not gonna allow this change

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/atmospherics/gasmixtures/gas_types.dm (hunk 1 above)
Comment (nicbn) [reply code/modules/atmospherics/gasmixtures/gas_types.dm]: Ok but can n2o canister be called nitrous canister at least?

**Comment by YakumoChen** [comment]
wouldn't nitrogen dioxide be NO2?

**Comment by nicbn** [comment]
Everyone confuses it with N2O.

**Comment by Qustinnus** [comment]
this is stupid

**Review by ChangelingRain — CHANGES_REQUESTED** [review]
please split the icons out of this pr.

**Comment by nicbn** [comment]
Canister PR: #32219

**Comment by AuroranAI** [comment]
>throws this idea in original pr
>ignored in pr
>try to push it anyways

Renaming it to nitrogen dioxide is bleh because that'll lead to people getting confused between n2o and no2.

Renaming stimulum to nikolayev gas is bad because it's a reference to a crappy creepypasta and obscures what the gas does. Stimulum obviously stimulates you. Without you telling me the name was related to the sleep experiment, i would have not gotten the connection.

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 2]
```diff
@@ -11,15 +11,15 @@
...
-#define STIMULUM_HEAT_SCALE					100000
-#define STIMULUM_FIRST_RISE					0.65
-#define STIMULUM_FIRST_DROP					0.065
-#define STIMULUM_SECOND_RISE				0.0009
-#define STIMULUM_ABSOLUTE_DROP				0.00000335
+#define STIMGAS_HEAT_SCALE					100000
+#define STIMGAS_FIRST_RISE					0.65
+#define STIMGAS_FIRST_DROP					0.065
+#define STIMGAS_SECOND_RISE					0.0009
+#define STIMGAS_ABSOLUTE_DROP				0.00000335
```
Comment (AuroranAI) [inline code/modules/atmospherics/gasmixtures/reactions.dm]: if you can't even be assed to change the full name of the variables to match up with how it used to be, why are you changing it?

**Comment by ChangelingRain** [comment]
Anyway, since the original PR maker who added the gasses, the atmos maintainer, and I all don't like it...

**Comment by Qustinnus** [comment]
thank you maintainer person

**Comment by nicbn** [comment]
The problem here is that browns gas is the name to another fucking gas.

**Comment by Qustinnus** [comment]
the bigger problem is that your gas names are super stupid :^)

**Comment by nicbn** [comment]
If you want I can keep stimulum as is, but come on, naming something with the name of another thing is not cool.

**Comment by Qustinnus** [comment]
then think of names that arent poo

**Comment by nicbn** [comment]
Nitrogen dioxide is super stupid? Lol say this to IUPAC

**Comment by Qustinnus** [comment]
it is in terms of a game, there would be 2 gasses with similar names that people will get switched around a fuckton. And thats not even the one I thought was the shittiest, the nikolayev gas or whatever is even worse.

**Comment by Bawhoppen** [comment]
Stimulum sounds dumb and gamey and brown gas is just weird

**Comment by Qustinnus** [comment]
yes but this is not an improvement. this just makes the problem worse.

**Comment by Bawhoppen** [comment]
Nikolayev gas definitely is an improvement from stimulum. It just sounds better. Though the nitrogen dioxide confusion thing is a real concern.

**Comment by nicbn** [comment]
That's why I called it nitro dioxide, is sounds less complicated I think.
Anyway it can be also called nitric gas.
I'd like to get this reopened and get some more feedback.

**Comment by duncathan** [comment]
@nicbn I will support a PR that renames brown gas to nitryl or to azote dioxide, but I'd like stimulum to stay as is. Either of those two names are fine imo so take your pick and ping me when you've done it. 

Renaming brown gas is good because 1) the name is awful and meaningless and 2) it conflicts with brown's gas, which it is not. I just want to be careful to avoid including the word "nitrogen" to prevent as much confusion as possible.

**Comment by nicbn** [comment]
Ok, gonna make a new PR to change it to nitryl (azote dioxide sounds too technical).

**Pull request closed without merging**